package.json

```diff
@@ -83,6 +83,11 @@
                 "peerDependencies": {
                     "@types/react": "^18.0.0"
                 }
+            },
+            "react-final-form": {
```

Comment:
We also use react-final-form in the Admin, where we have React 17. Could this lead to errors?

Reply:
I have no idea... 😅 

@thomasdax98 Is there a plan to make the react versions consistent? Or is there a reason for this?